### PR TITLE
Updated AMP overview screen (ADBP-1689)

### PR DESCRIPTION
**Pull request by sk101dio**
### Why

UI changes of AMP overview screen

Link to Jira ticket: https://101digital.atlassian.net/browse/ADBP-1689

### CODE REVIEW

File: src/components/adb-join-member-plus-confirm/index.tsx [hunk 1]
```diff
@@ -77,7 +88,14 @@ const ADBJoinMemberPlusConfirmComponent = (
       </View>
       <View style={styles.bottomSection}>
         <Button
-          label={'Confirm'}
+          label={'Go to Home'}
+          onPress={goToHome}
+          background={'transparent'}
+          labelColor={'#1B1B1B'}
```
Comment (sharmal33) [inline src/components/adb-join-member-plus-confirm/index.tsx:94]: can you use theme colors

File: src/components/adb-join-member-plus-confirm/index.tsx [hunk 2]
```diff
@@ -77,7 +88,14 @@ const ADBJoinMemberPlusConfirmComponent = (
       </View>
       <View style={styles.bottomSection}>
         <Button
-          label={'Confirm'}
+          label={'Go to Home'}
+          onPress={goToHome}
```
Comment (sharmal33) [inline src/components/adb-join-member-plus-confirm/index.tsx:92]: get all static content from translations file

File: src/components/adb-join-member-plus-confirm/index.tsx [hunk 3]
```diff
@@ -77,7 +88,14 @@ const ADBJoinMemberPlusConfirmComponent = (
...
+          background={'transparent'}
+          labelColor={'#1B1B1B'}
+        />
+        <View style={styles.buttonMargin}/>
+        <Button
+          label={'Open Savings Account'}
```
Comment (sharmal33) [inline src/components/adb-join-member-plus-confirm/index.tsx:98]: get this string from translations file

**Review by sharmal33 — CHANGES_REQUESTED** [review]

File: src/components/adb-join-member-plus-confirm/index.tsx (hunk 1 above)
Comment (sk101dio) [reply src/components/adb-join-member-plus-confirm/index.tsx:96]: Actually I used this color from Register button of Onboarding screen https://github.com/101digital/account-origination-component/blob/ADBP-1689/src/components/adb-onboarding-slides/index.tsx 

`<Button
            label={i18n.t('common.lbl_register') ?? 'Register'}
            onPress={onRegister}
            background={'transparent'}
            labelColor={'#1B1B1B'}
          />`

https://www.figma.com/file/dymn49kn1xZhNBJJcXPDMW/ADB-%F0%9F%94%B6-AEON-DigiBank-(EAJ00311)---GQ18102022?node-id=20056-280269&t=pIeOpepPpZqTHBv9-4

Currently I am not sure about the theme color, please point me to the theme color I should use..

File: src/components/adb-join-member-plus-confirm/index.tsx (hunk 3 above)
Comment (sk101dio) [reply src/components/adb-join-member-plus-confirm/index.tsx:100]: sure, updating code accordingly

File: src/components/adb-join-member-plus-confirm/index.tsx (hunk 2 above)
Comment (sk101dio) [reply src/components/adb-join-member-plus-confirm/index.tsx:94]: sure, updating code accordingly

**Pull request merged**